File: 30 Days of Code/Day02: Operators/operators.swift
let mealCost = Double(readLine()!)!
let tipPercent = Double(readLine()!)!
let taxPercent = Double(readLine()!)!

var totalCost = mealCost * (100.0 + tipPercent + taxPercent) / 100.0
totalCost.round()
let answer = Int(totalCost)
print("The total meal cost is \(answer) dollars.")

File: 30 Days of Code/Day02: Operators/operators.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        double meal_cost = in.nextDouble();
        int tip_percent = in.nextInt();
        int tax_percent = in.nextInt();
        in.close();

        double total_cost = meal_cost * (100 + tip_percent + tax_percent) / 100;
        long rounded_cost = Math.round(total_cost);
        String answer = String.format("The total meal cost is %s dollars.", rounded_cost);
        System.out.println(answer);
    }
}
